camel-k: simple integrations without a web part have a startup WARNING

## This is a...

<pre><code>
[ ] Feature request
[ ] Regression (a behavior that used to work and stopped working in a new release)
[X ] Bug report  <!-- Please search GitHub for a similar issue or PR before submitting -->
[ ] Documentation issue or request
</code></pre>

## Description
Using camel-k runtime, if you start a simple integration like from timer to log, this WARNING is logged when the integration pod starts:
```
2019-05-15 20:55:12.806 WARN  [main] AnnotationTypeConverterLoader - Ignoring converter type: org.apache.camel.http.common.HttpConverter as a dependent class could not be found: java.lang.NoClassDefFoundError: javax/servlet/ServletInputStream
java.lang.NoClassDefFoundError: javax/servlet/ServletInputStream
	at java.lang.Class.getDeclaredMethods0(Native Method) ~[?:1.8.0_191]
	at java.lang.Class.privateGetDeclaredMethods(Class.java:2701) ~[?:1.8.0_191]
	at java.lang.Class.getDeclaredMethods(Class.java:1975) ~[?:1.8.0_191]
	at org.apache.camel.impl.converter.AnnotationTypeConverterLoader.loadConverterMethods(AnnotationTypeConverterLoader.java:262) ~[org.apache.camel.camel-core-2.21.0.fuse-740010.jar:2.21.0.fuse-740010]
	at org.apache.camel.impl.converter.AnnotationTypeConverterLoader.load(AnnotationTypeConverterLoader.java:130) ~[org.apache.camel.camel-core-2.21.0.fuse-740010.jar:2.21.0.fuse-740010]
	at org.apache.camel.impl.converter.BaseTypeConverterRegistry.loadTypeConverters(BaseTypeConverterRegistry.java:640) ~[org.apache.camel.camel-core-2.21.0.fuse-740010.jar:2.21.0.fuse-740010]
	at org.apache.camel.impl.converter.DefaultTypeConverter.doStart(DefaultTypeConverter.java:62) ~[org.apache.camel.camel-core-2.21.0.fuse-740010.jar:2.21.0.fuse-740010]
	at org.apache.camel.support.ServiceSupport.start(ServiceSupport.java:61) ~[org.apache.camel.camel-core-2.21.0.fuse-740010.jar:2.21.0.fuse-740010]
	at org.apache.camel.util.ServiceHelper.startService(ServiceHelper.java:75) ~[org.apache.camel.camel-core-2.21.0.fuse-740010.jar:2.21.0.fuse-740010]
	at org.apache.camel.impl.DefaultCamelContext.doAddService(DefaultCamelContext.java:1490) ~[org.apache.camel.camel-core-2.21.0.fuse-740010.jar:2.21.0.fuse-740010]
	at org.apache.camel.impl.DefaultCamelContext.addService(DefaultCamelContext.java:1450) ~[org.apache.camel.camel-core-2.21.0.fuse-740010.jar:2.21.0.fuse-740010]
	at org.apache.camel.impl.DefaultCamelContext.getTypeConverter(DefaultCamelContext.java:2571) ~[org.apache.camel.camel-core-2.21.0.fuse-740010.jar:2.21.0.fuse-740010]
	at org.apache.camel.util.IntrospectionSupport.setProperty(IntrospectionSupport.java:652) ~[org.apache.camel.camel-core-2.21.0.fuse-740010.jar:2.21.0.fuse-740010]
	at io.syndesis.integration.runtime.handlers.HandlerCustomizer.setProperties(HandlerCustomizer.java:86) ~[io.syndesis.integration.integration-runtime-1.7-SNAPSHOT.jar:1.7-SNAPSHOT]
	at io.syndesis.integration.runtime.handlers.ConnectorStepHandler.handle(ConnectorStepHandler.java:124) ~[io.syndesis.integration.integration-runtime-1.7-SNAPSHOT.jar:1.7-SNAPSHOT]
	at io.syndesis.integration.runtime.IntegrationRouteBuilder.configureFlow(IntegrationRouteBuilder.java:145) ~[io.syndesis.integration.integration-runtime-1.7-SNAPSHOT.jar:1.7-SNAPSHOT]
	at io.syndesis.integration.runtime.IntegrationRouteBuilder.configure(IntegrationRouteBuilder.java:112) ~[io.syndesis.integration.integration-runtime-1.7-SNAPSHOT.jar:1.7-SNAPSHOT]
	at org.apache.camel.builder.RouteBuilder.checkInitialized(RouteBuilder.java:462) ~[org.apache.camel.camel-core-2.21.0.fuse-740010.jar:2.21.0.fuse-740010]
	at org.apache.camel.builder.RouteBuilder.configureRoutes(RouteBuilder.java:402) ~[org.apache.camel.camel-core-2.21.0.fuse-740010.jar:2.21.0.fuse-740010]
	at org.apache.camel.builder.RouteBuilder.addRoutesToCamelContext(RouteBuilder.java:383) ~[org.apache.camel.camel-core-2.21.0.fuse-740010.jar:2.21.0.fuse-740010]
	at org.apache.camel.impl.DefaultCamelContext$1.call(DefaultCamelContext.java:1027) ~[org.apache.camel.camel-core-2.21.0.fuse-740010.jar:2.21.0.fuse-740010]
	at org.apache.camel.impl.DefaultCamelContext$1.call(DefaultCamelContext.java:1024) ~[org.apache.camel.camel-core-2.21.0.fuse-740010.jar:2.21.0.fuse-740010]
	at org.apache.camel.impl.DefaultCamelContext.doWithDefinedClassLoader(DefaultCamelContext.java:3270) ~[org.apache.camel.camel-core-2.21.0.fuse-740010.jar:2.21.0.fuse-740010]
	at org.apache.camel.impl.DefaultCamelContext.addRoutes(DefaultCamelContext.java:1024) ~[org.apache.camel.camel-core-2.21.0.fuse-740010.jar:2.21.0.fuse-740010]
	at org.apache.camel.k.listener.RoutesConfigurer.load(RoutesConfigurer.java:73) ~[org.apache.camel.k.camel-k-runtime-core-0.3.2.jar:0.3.2]
	at org.apache.camel.k.listener.RoutesConfigurer.accept(RoutesConfigurer.java:45) ~[org.apache.camel.k.camel-k-runtime-core-0.3.2.jar:0.3.2]
	at org.apache.camel.k.listener.AbstractPhaseListener.accept(AbstractPhaseListener.java:31) ~[org.apache.camel.k.camel-k-runtime-core-0.3.2.jar:0.3.2]
	at org.apache.camel.k.jvm.ApplicationRuntime$MainListenerAdapter.lambda$configure$2(ApplicationRuntime.java:113) ~[org.apache.camel.k.camel-k-runtime-jvm-0.3.2.jar:0.3.2]
	at java.lang.Iterable.forEach(Iterable.java:75) [?:1.8.0_191]
	at org.apache.camel.k.jvm.ApplicationRuntime$MainListenerAdapter.configure(ApplicationRuntime.java:113) [org.apache.camel.k.camel-k-runtime-jvm-0.3.2.jar:0.3.2]
	at org.apache.camel.main.MainSupport.postProcessCamelContext(MainSupport.java:618) [org.apache.camel.camel-core-2.21.0.fuse-740010.jar:2.21.0.fuse-740010]
	at org.apache.camel.main.MainSupport.postProcessContext(MainSupport.java:550) [org.apache.camel.camel-core-2.21.0.fuse-740010.jar:2.21.0.fuse-740010]
	at org.apache.camel.k.adapter.Main.doStart(Main.java:46) [org.apache.camel.k.camel-k-adapter-camel-2-0.3.2.jar:0.3.2]
	at org.apache.camel.support.ServiceSupport.start(ServiceSupport.java:61) [org.apache.camel.camel-core-2.21.0.fuse-740010.jar:2.21.0.fuse-740010]
	at org.apache.camel.main.MainSupport.run(MainSupport.java:170) [org.apache.camel.camel-core-2.21.0.fuse-740010.jar:2.21.0.fuse-740010]
	at org.apache.camel.k.jvm.ApplicationRuntime.run(ApplicationRuntime.java:66) [org.apache.camel.k.camel-k-runtime-jvm-0.3.2.jar:0.3.2]
	at org.apache.camel.k.jvm.Application.main(Application.java:52) [org.apache.camel.k.camel-k-runtime-jvm-0.3.2.jar:0.3.2]
Caused by: java.lang.ClassNotFoundException: javax.servlet.ServletInputStream
	at java.net.URLClassLoader.findClass(URLClassLoader.java:382) ~[?:1.8.0_191]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424) ~[?:1.8.0_191]
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:349) ~[?:1.8.0_191]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357) ~[?:1.8.0_191]
... 37 more
```

--- a/app/integration/runtime/pom.xml
+++ b/app/integration/runtime/pom.xml
@@ -91,6 +91,11 @@
       <artifactId>camel-http-common</artifactId>
     </dependency>
     <dependency>
+      <groupId>javax.servlet</groupId>
+      <artifactId>javax.servlet-api</artifactId>
+      <scope>runtime</scope>
+    </dependency>
+    <dependency>
       <groupId>com.github.spotbugs</groupId>
       <artifactId>spotbugs-annotations</artifactId>
     </dependency>

